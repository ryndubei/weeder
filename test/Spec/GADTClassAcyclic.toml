roots = [ "Spec.GADTClassAcyclic.GADTClassAcyclic.Foo" ]

type-class-roots = false

unused-types = true

root-instances = []

root-classes = []
