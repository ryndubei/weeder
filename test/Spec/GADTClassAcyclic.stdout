test/Spec/GADTClassAcyclic/GADTClassAcyclic.hs:6: MkFoo
test/Spec/GADTClassAcyclic/GADTClassAcyclic.hs:8: Bar
